variables.tf

```diff
@@ -311,3 +311,12 @@ variable "eks_oidc_root_ca_thumbprint" {
   description = "Thumbprint of Root CA for EKS OIDC, Valid until 2037"
   default     = "9e99a48a9960b14926bb7f3b02e22da2b0ab7280"
 }
+
+variable "encryption_config" {
```

Comment:
Can you please rename this variable into `cluster_encryption_config` ?

Reply:
Certainly. I have just pushed changes to that effect.